src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -60,21 +61,48 @@ def generate_definition(class_name: str) -> str:
 def _generate_declaration_for_class(class_: Any) -> List[str]:
     """ Return all attributes for the given Class with CaseDescription definition. """
     class_fields = attr.fields_dict(class_)
-    lines = [f"{INDENT*2}class {class_.__name__}"]
-    lines.extend(_get_declaration(class_fields, LIST_OF_CASE_ATTRIBUTES))
-    return lines
+    return [
+        f"{INDENT*2}class {class_.__name__}",
+        *_get_declaration(class_fields, LIST_OF_CASE_ATTRIBUTES),
+    ]
 
 
 def _generate_declaration_for_schema(class_: Any) -> List[str]:
     """ Return all attributes for the given Class using ALFACase schema definition. """
     class_fields = attr.fields_dict(class_)
-    lines = _get_declaration(class_fields, LIST_OF_CASE_SCHEMAS)
-    return lines
+    return _get_declaration(class_fields, LIST_OF_CASE_SCHEMAS, is_schema=True)
+
+
+def _get_default_value_of_case(value: attr.Attribute) -> str:
+    """ Return the default value of a CaseDescription. """
+
+    if value.default == attr.NOTHING:
+        return ""
+
+    default_value_string = " = "
+    if isinstance(value.default, enum.Enum):
+        default_value_string += (
+            f"{value.default.__class__.__name__}.{value.default.name}"
+        )
+    elif isinstance(value.default, attr.Factory):
+        if value.default.factory is dict:
+            default_value_string += "{}"
+        elif value.default.factory is list:
+            default_value_string += "[]"
+        elif is_attrs(value.default):
```

Comment:
This `if`/`elif` chain doesn't have an `else` clause, which will produce a strange `" = "` string when we get a factory which doesn't match any of the ones being checked here. How about `else: default_value_string += "UNKNOWN FACTORY"` or something like this? While an ugly name, it will help us spot those in the docs and update accordingly.

Reply:
Done